src/includes/performance/configure-sample-rate/java.spring-boot.mdx

```diff
@@ -8,7 +8,7 @@ sentry.traces-sample-rate=0.3
 
 Or through providing a bean of type `SentryOptions#TracesSamplerCallback`:
 
-```java {tabTitle:Java}
+```java
```

Comment:
why are we removing all the `tabTitle`?

Reply:
in this case doesn't change anything in the rendered html but enables code highlighting in IDE